src/Microsoft.Framework.PackageManager/Scripts/ScriptExecutor.cs

```diff
@@ -15,6 +15,13 @@ public class ScriptExecutor
     {
         private static readonly string ErrorMessageTemplate = "The '{0}' script failed with status code {1}.";
 
+        public ScriptExecutor(IReport report = null)
+        {
+            Report = report;
+        }
+
+        public IReport Report { get; set; }
```

Comment:
To make it work with `RestoreCommand` class, we need this public setter. `RestoreCommand` cannot finalize the value of `Reports` when it is constructing `ScriptExecutor`, so setting `IReport` with constructor is not enough.

We may need to redesign `RestoreCommand` to get rid of this but it's a separate topic.